practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -51,9 +57,56 @@ const FormValidation = (props) => {
     [setErrors]
   );
 
+  // Reset form value
+  const resetForm = () => {
+    setFormData({
+      id: '',
```

Comment:
do you fill the id in form?

Reply:
I set up the Id in the form to perform the update function later
I will check the condition: if Id is present, the update feature will be performed, if not, the create feature will be performed
![image](https://github.com/datdragon777/React-Training/assets/74806826/de593fe6-c9b1-4b4e-ac58-7d0fa79adb6a)